dessia_common/workflow/core.py

```diff
@@ -2186,6 +2186,67 @@ def _method_jsonschemas(self):
         jsonschemas['run_again']['classes'] = ["dessia_common.workflow.WorkflowRun"]
         return jsonschemas
 
+    def to_script(self):
+        """
+        Computes a script representing the workflowrun.
+        """
+        # TODO : don't take into account yet TypedVariable.
+        # TODO : take into account the type of inputs (type_)
+        workflow_script = self.workflow.to_script()
+        workflow_script_import = self.workflow._to_script().imports
+        input_str = ""
+        default_value = ""
+        add_import = ""
+        var_1 = "block_{}.inputs[{}]"
```

Comment:
We should set this variable inside the loop to avoid the .format() stuff and use fstrings instead.

Also var_1 is a poor naming and should be improved. Why not variable or even just var ?